onnx2kerastl/operation_layers.py

```diff
@@ -223,6 +223,52 @@ def target_layer(x, axis=params['axes'], keepdims=params['keepdims']):
     lambda_func[keras_name] = target_layer
 
 
+def convert_reduce_min(node, params, layers, lambda_func, node_name, keras_name):
+    """
+    Convert reduce max.
+    :param node: current operation node
+    :param params: operation attributes
+    :param layers: available keras layers
+    :param lambda_func: function for keras Lambda layer
+    :param node_name: internal converter name
+    :param keras_name: resulting layer name
+    :return: None
+    """
+    if params.get("axes") is not None: #opset 13
+        axes = params.get("axes")
+    elif len(node.input) == 2:
+        axes = layers.get(node.input[1])
+    noop_with_empty_axes = bool(params.get("noop_with_empty_axes", False))
+    keepdims = params.get("keepdims", True)
+    if not noop_with_empty_axes:
+        layers[node_name] = tf.math.reduce_min(layers[node.input[0]], axis=axes, keepdims=keepdims)
+    else:
+        layers[node_name] = layers[node.input[0]]
```

Comment:
> Defines behavior if 'axes' is empty. Default behavior with 'false' is to reduce all axes. When axes is empty and this attribute is set to true, input tensor will not be reduced,and the output tensor would be equivalent to input tensor.

from my look at the documentation the condition should be `not noop_with_empty_axes and axes is None`

Reply:
Can you also please convert the if condition from `not noop_with_empty_axes` to `if noop_with_empty_axes`, It feels confusing 😄